.travis.yml

```diff
@@ -21,7 +21,7 @@ addons:
 
 before_install:
   - go get -u github.com/kardianos/govendor && govendor sync -v
-  - go get -u github.com/alecthomas/gometalinter && gometalinter --install
+  - if [ "go version go1.8.7 linux/amd64" != "$(go version)" && "go version go1.7.6 linux/amd64" != "$(go version)" ]; then go get -u github.com/alecthomas/gometalinter && gometalinter --install; fi
```

Comment:
This if isn't quite right and Travis ends up failing to execute it. I think one way to fix it is to surround each case with `[ ]` brackets, i.e., `if [ "go version go1.8.7 linux/amd64" != "$(go version)" ] && [ "go version go1.7.6 linux/amd64" != "$(go version)" ]; then ...`